docs/prd/code-generation.md

```
@@ -0,0 +1,610 @@
+
+      default_tags {
+        tags = {
+          Environment = "{{ .vars.environment }}"
+          Component   = "{{ .atmos_component }}"
+        }
+      }
+    }
+```
+
+**YAML Block Scalar Reference:**
+| Style | Syntax | Behavior |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)